vvm/install.py

```diff
@@ -162,6 +163,7 @@ def _get_releases(headers: Optional[Dict]) -> Dict:
     return data.json()
 
 
+@functools.lru_cache(maxsize=1, typed=False)
```

Comment:
this can be invalidated though by installing new vyper versions right?

Reply:
This is not about installed versions but installable versions.
When a new Vyper version is released, the application would need to be restarted, given the cache is in memory.
Otherwise, the user would need to call `get_installable_vyper_versions.cache_clear()`

Reply:
in that case why limit the cache size to 1? also, let's add a comment
```suggestion
# When a new Vyper version is released, the application would need to be restarted, given the cache is in memory.
# Otherwise, the user would need to call get_installable_vyper_versions.cache_clear()
@functools.lru_cache
```

Reply:
also i'm not sure a function which takes dict as a param works for lru_cache --

https://docs.python.org/3/library/functools.html#functools.lru_cache
> Since a dictionary is used to cache results, the positional and keyword arguments to the function must be [hashable](https://docs.python.org/3/glossary.html#term-hashable).